inspirehep/modules/references/processors.py

```diff
@@ -90,11 +90,10 @@ def _is_arxiv(obj):
     """Small checker for arXiv report numbers."""
     # Workaround until this issue:
     # https://github.com/inveniosoftware/idutils/issues/14 is solved.
-
     arxiv_test = obj.split()
     if not arxiv_test:
         return False
-    return idutils.is_arxiv(arxiv_test[0])
+    return idutils.is_arxiv(arxiv_test[0]) or False
```

Comment:
I wouldn't say this is a bug in `idutils`; the expected pythonic use
```python
if is_arxiv(foo):
  ...
```
is indifferent to having `False` or `None`.

👍  instead for wrapping in `bool`.